solidity/test/altbn128/TestAltBn128.js

```diff
@@ -1,8 +1,8 @@
 const { expectRevert } = require("@openzeppelin/test-helpers")
+const { contract } = require("@openzeppelin/test-environment")
+const AltBn128Stub = contract.fromArtifact("AltBn128Stub")
 
-const AltBn128Stub = artifacts.require("./stubs/AltBn128Stub.sol")
-
-contract("AltBn128", () => {
+describe("TestAltBn128", () => {
```

Comment:
We should have `describe('AltBn128'` here as we are describing `AltBn128` behaviour.